CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contribution guide
+
+All contributions are welcome! :)
+
+If you would like to contribute anything, fork the repo and open a Pull Request (PR).
+
+## Recommended git workflow
+
+The best way to ensure that git history is not jumbled up too much is to add an `upstream` remote:
+
+```
+$ git clone ...your_fastats_fork_url...
+$ cd fastats
+$ git remote add upstream https://github.com/fastats/fastats
+```
+
+Then you can fetch from `upstream` remote and create new features on your fork easily:
+
+```
+$ git fetch upstream
+$ git checkout -b my_awesome_branch upstream/master
+```
+
+
+## Issues
+
+Issues are turned off forever.  We prefer Pull Requests for everything.
```

Comment:
I'm quite fixed on this one, just like @dwillmer - issues are just evil.

Reply:
+1